en/migrations.rst

```diff
@@ -110,38 +110,80 @@ Creating Migrations
 
 Migration files are stored in the **config/Migrations** directory of your
 application. The name of the migration files are prefixed with the date in
-which they were created, in the format **YYYYMMDDHHMMSS_my_new_migration.php**.
+which they were created, in the format **YYYYMMDDHHMMSS_my_new_migration.php**::
+
+        -rw-rw-r-- 1 root root 1066 Aug 21 22:05 20150822030519_create_articles.php
 
-The easiest way of creating a migrations file is by using the command line.
-Let's imagine that you'd like to add a new ``products`` table::
+The easiest way to create a migrations file is by using the :doc:`/bake/usage` CLI command.
+The following ``Bake`` command would create a migration to add a ``products`` table::
 
-        bin/cake bake migration CreateProducts name:string description:text created modified
+        $ bin/cake bake migration CreateProducts name:string description:text created modified
+        
+        Welcome to CakePHP v3.1.7 Console
+        ---------------------------------------------------------------
+        App : src
+        Path: /home/user/Work/php/cakeblog/src/
+        PHP : 5.5.28-1+deb.sury.org~precise+1
+        ---------------------------------------------------------------
+        
+        Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121163249_CreateProducts.php
+        Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121163249_CreateProducts.php`
 
 .. note::
 
-        You may also choose to use the underscore_form as migration label i.e.::
+        You can also use the ``underscore_form`` as the name for your migrations
+        i.e. create_products::
 
-            bin/cake bake migration create_products name:string description:text created modified
+                $ bin/cake bake migration create_products name:string description:text created modified
+            
+                Welcome to CakePHP v3.0.13 Console
+                ---------------------------------------------------------------
+                App : src
+                Path: /home/user/Work/php/cakeblog/src/
+                ---------------------------------------------------------------
+                
+                Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php
+                Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php`
```

Comment:
This is not true anymore as of v1.5.2 of the migrations plugin. It will always create a camelized version of the slug provided.
See https://github.com/cakephp/migrations/issues/152 and https://github.com/cakephp/migrations/pull/175